pkg/domain/entity/task.go

```diff
@@ -0,0 +1,16 @@
+package entity
+
+import (
+	"time"
+)
+
+type Task struct {
+	ID          string
+	CreatedAt   time.Time
+	UpdatedAt   time.Time
+	DeletedAt   time.Time
```

Comment:
削除してない状態で`DeletedAt`は`null`なので
```suggestion
	DeletedAt   *time.Time
```

Reply:
https://github.com/CATechAccel/hack-ios-server/pull/14/commits/3a2b765650ada3e210a96d7178b9c6651013582f
ここで修正しましたが，これで良さそうですか？